### PR TITLE
Feature title and message html based on previous PR

### PR DESCRIPTION
**Pull request by bholland314**
Ex. for message
<p style='border: black solid 2px;'><span>Test <mark>Message</mark><span></p>

### CODE REVIEW

File: src/Sotsera.Blazor.Toaster/Core/Models/CommonToastOptions.cs [hunk 1]
```diff
@@ -10,6 +10,10 @@ public abstract class CommonToastOptions
         /// </summary>
         public string ToastClass { get; set; } = Defaults.Classes.Toast;
 
+        /// <summary>
+        /// This variable applies to the title and message. If false HTML will be implemented, else HTML markup will be shown. />.
```
Comment (ghidello) [inline src/Sotsera.Blazor.Toaster/Core/Models/CommonToastOptions.cs]: Do you think we can clarify a little more the behaviour of this toggle option? Maybe using something like this:
```suggestion
        /// This variable applies to the title and message. If false the rendered HTML will be shown, else the escaped HTML markup will be shown. 
```

File: src/Sotsera.Blazor.Toaster/Core/Models/CommonToastOptions.cs [hunk 2]
```diff
@@ -10,6 +10,10 @@ public abstract class CommonToastOptions
...
+        /// </summary>
+        public bool EscapeHtml { get; set; } = false;
```
Comment (ghidello) [inline src/Sotsera.Blazor.Toaster/Core/Models/CommonToastOptions.cs]: I think that, for a security concern, the text should be escaped by default: what do you think about it?

```suggestion
        public bool EscapeHtml { get; set; } = true;
```

File: src/Sotsera.Blazor.Toaster/Core/ToastElement.razor [hunk 3]
```diff
@@ -4,21 +4,35 @@
 *@
 @inherits ToastElementModel
 @Css
-<div class="@Toast.State.ToastClass" style="@Toast.State.AnimationStyle" @onmouseenter="MouseEnter" @onmouseleave="MouseLeave" @onclick="Clicked">
```
Comment (ghidello) [inline src/Sotsera.Blazor.Toaster/Core/ToastElement.razor:7]: I must review the toast class and see why I made the Options class available from both the Options property and the State.Options property.. I changed the project so many times during the asp.net beta releases that I really messed up here. I'll open an issue for this and we'll have to uniform the Options usage in this component.

**Review by ghidello — CHANGES_REQUESTED** [review]
This looks very nice to me. Thanks! I added a coule of comments to discuss with you and so I also learnt how to propose a pull request change: thanks for this too! :smile:

File: src/Sotsera.Blazor.Toaster/Core/ToastElement.razor (hunk 3 above)
Comment (bholland314) [reply src/Sotsera.Blazor.Toaster/Core/ToastElement.razor:7]: Sounds like a good idea.

File: src/Sotsera.Blazor.Toaster/Core/Models/CommonToastOptions.cs (hunk 2 above)
Comment (bholland314) [reply src/Sotsera.Blazor.Toaster/Core/Models/CommonToastOptions.cs]: Not sure what happened the first time I commented, but it seemed to go away. 
My comment is this: I followed the same convention as Toastr.js.

**Review by ghidello — APPROVED** [review]

**Pull request merged**